dragonchain/webserver/routes/verifications.py

```diff
@@ -39,3 +42,8 @@ def get_verification_v1(block_id: str, **kwargs) -> Tuple[str, int, Dict[str, st
 @request_authorizer.Authenticated(api_resource="verifications", api_operation="read", api_name="get_pending_verifications")
 def get_pending_verifications_v1(block_id: str, **kwargs) -> Tuple[str, int, Dict[str, str]]:
     return helpers.flask_http_response(200, verifications.get_pending_verifications_v1(block_id))
+
+
+@request_authorizer.Authenticated(api_resource="verifications", api_operation="read", api_name="get_subsequent_verifications")
```

Comment:
`get_subsequent_verifications` will need to be added to the endpoint map in api_key_model, the permission document schema, and the table in the documentation on the permissioning page.